ARM.CMSIS.pdsc

```diff
@@ -668,6 +682,24 @@
       </files>
     </component>
 
+    <component Cclass="CMSIS" Cgroup="CORE" Cversion="1.0.0"  condition="ARMv8-A Device" >
+      <description>CMSIS-CORE for Cortex-A</description>
+      <files>
+        <!-- CPU independent -->
+        <file category="doc"     name="CMSIS/Documentation/html/Core_A/index.html"/>
+        <file category="include" name="CMSIS/Core/Include/"/>
+      </files>
+    </component>
+
+    <component Cclass="CMSIS" Cgroup="CORE" Cversion="1.0.0"  condition="ARMv7-R Device" >
+      <description>CMSIS-CORE for Cortex-R</description>
+      <files>
+        <!-- CPU independent -->
+        <file category="doc"     name="CMSIS/Documentation/html/Core_R/index.html"/>
```

Comment:
I don't think that we have this file, do we?

Reply:
No, we don't have the file, I have commented it out. So we have a reminder that the documentation still needs to be added.